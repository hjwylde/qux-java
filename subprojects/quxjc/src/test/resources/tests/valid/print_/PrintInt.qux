package valid.print_

void main([str] args):
    print 0
    print -0
    print 5
    print -5
    print 0b010
    print -0b010
    print 0o70
    print -0o70
    print 0xa0
    print -0xa0

