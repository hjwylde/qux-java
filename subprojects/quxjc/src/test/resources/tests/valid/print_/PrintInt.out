0
0
5
-5
2
-2
56
-56
160
-160
